hcert_spec.md

```diff
@@ -1,6 +1,6 @@
 # Electronic Health Certificate Specification
 
-Version 1.0.5, 2021-04-18
+Version 1.0.6-PRE, 2021-04-XX
```

Comment:
Need to set 1.0.6 in `next` and update changelog first, as you can't commit once merged.